assets/js/blocks/price-filter/block.js

```diff
@@ -95,33 +110,30 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 	// Updates the query based on slider values.
 	const onSubmit = useCallback(
 		( newMinPrice, newMaxPrice ) => {
+			// Using snake_case, just for ease of assignment to URL params.
```

Comment:
You didn't end up using `snake_case` though, so this comment might not be required :P

Reply:
Good catch, I decided against `snake_case` after using it in my first approach but obviously forgot to remove the comment!